astra-sim/workload/Workload.cc

```diff
@@ -252,19 +252,24 @@ void Workload::issue_comm(shared_ptr<Chakra::ETFeederNode> node) {
       collective_comm_node_id_map[fp->my_id] = node->id();
       collective_comm_wrapper_map[fp->my_id] = fp;
       fp->set_notifier(this, EventType::CollectiveCommunicationFinished);
+
```

Comment:
Let's avoid any unnecessary changes like blank lines.

Reply:
I put that blank lines because the other if-statement did that in the same way. 
Do you think we need to remove all those as well?